womtool/src/test/resources/validate/wdl_draft3/invalid/bad_local_import/error.txt

```diff
@@ -0,0 +1 @@
+Import file not found: not/a/workflow.wdl
```

Comment:
This message looks wrong to me for draft-3.

Luckily [I have a PR](https://github.com/broadinstitute/cromwell/pull/3987) which provides a draft-3 regression test for this instead, so I'd just remove the draft-3 version